### PR TITLE
#170714853 add booking endpoints

### PR DESCRIPTION
**Pull request by shaazk**
What does this PR do?
- adds booking endpoint
- adds a view all bookings endpoint
- adds database scripts for the two endpoints

Description of Task to be completed?
- Adds booking endpoints

How should this be manually tested?

After cloning the repo, start the server and go to your postman and use the paths below
- to add a new booking under POST, use "http://localhost:6000/api/bookings" and fill in the required fields in the body
- to view all bookings under GET, use "http://localhost:6000/api/bookings"and fill in the required fields in the body

What are the relevant pivotal tracker stories?
#170739200

### CODE REVIEW

File: src/controller/booking.controller.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import Booking from '../model/booking.models'
+
+class BookingController {
+    static async createBooking(req, res) {
+        try {
+          const createdBooking = await Booking.create(req.body);
```
Comment (Elvis-rugamba) [inline src/controller/booking.controller.js:6]: Validate your data

**Comment by Skemc** [comment]
well organized keep up the spirit

**Review by Skemc — COMMENTED** [review]
good work

File: src/controller/booking.controller.js (hunk 1 above)
Comment (shaazk) [reply src/controller/booking.controller.js:6]: done. thanks

**Pull request merged**